barretenberg/cpp/src/barretenberg/commitment_schemes/zeromorph/zeromorph.test.cpp

```diff
@@ -1,576 +1,579 @@
-#include "zeromorph.hpp"
-#include "../commitment_key.test.hpp"
-#include "barretenberg/commitment_schemes/ipa/ipa.hpp"
-#include "barretenberg/commitment_schemes/kzg/kzg.hpp"
-#include "barretenberg/transcript/transcript.hpp"
-
-#include <gtest/gtest.h>
-
-namespace bb {
-
-template <class PCS> class ZeroMorphTest : public CommitmentTest<typename PCS::Curve> {
-  public:
-    using Curve = typename PCS::Curve;
-    using Fr = typename Curve::ScalarField;
-    using Polynomial = bb::Polynomial<Fr>;
-    using Commitment = typename Curve::AffineElement;
-    using GroupElement = typename Curve::Element;
-    using VerifierAccumulator = typename PCS::VerifierAccumulator;
-    using ZeroMorphProver = ZeroMorphProver_<PCS>;
```

Comment:
Tried to engineer the diffs more but this is the best I can do, basically we used to have two test classes, for standard Zeromorph and Zeromorph with concatenation start. I tried to isolate the shared functionality into separate functions so the inputs they have in common are generated via the same code paths. Happy to hear suggestions of how to clean this more